cairo/src/Slay/Cairo.hs

```diff
@@ -116,6 +116,7 @@ instance RenderElement g (PrimCurve g) where
           (x + fst pEnd)   (y + snd pEnd)
     Cairo.moveTo (x + x1) (y + y1)
     curveThrough p1 p2 p3
+    Cairo.setLineWidth $ realToFrac (getG width)
```

Comment:
It's not that important, but I'd still prefer to see `toSigned` rather than `realToFrac` here, because it'll be more clear that we're not losing precision or anything.

Reply:
I see , ok